resources/weapons/rockets/AGR-20 M282.yaml

```diff
@@ -0,0 +1,5 @@
+name: LAU-131 pod - 7 x 2.75\" Hydra, Laser Guided Rkts M282, MPP APKWS
+year: 2012
+fallback: AGM-65F

```

Comment:
I wonder if this should fall back to the E/L maverick, and maybe to a double or triple rack? Hard to say about the second given the huge weight difference.

Reply:
That makes sense. The E/L maverick will be included in the next batch, probably early next week when I get time. I can then update this to fallback to it or whatever else makes sense.

Reply:
We can also merge this as-is and get the update later.

Reply:
That sounds like a plan. Thank you